ner_v1/detectors/textual/text/text_detection_model.py

```diff
@@ -98,11 +121,19 @@ def detect_entity(self, text, **kwargs):
             crf_original_texts = crf_model.detect_entity(text=text)
 
         values, original_texts = super(TextModelDetector, self).detect_entity(text, **kwargs)
-
-        text_entity_verified_values, original_texts = self.combine_results(values=values,
-                                                                           original_texts=original_texts,
-                                                                           crf_original_texts=crf_original_texts)
-        self.text_entity_values, self.original_texts = text_entity_verified_values, original_texts
+        if values and type(values[0]) is list:
```

Comment:
Revert these changes and move all this bulk implementation to `detect_entity_bulk()`. Do not call CrfDetetection in that method as we don't have a plan to add batch inference on CRF yet.
When calling `self.combine_results` in `detect_entity_bulk()`, pass `crf_original_texts=[]`

Do not assign anything to self.text_entity_values and self.original_texts, leave them untouched

Reply:
Done